scripts/build-operator

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source $(dirname $0)/lib/debug_functions
+
+cd $(dirname $0)
```

Comment:
w00t?

Reply:
Yes, because you can be calling this script from other directories.

I will squash this to the other "cd" down L9